crates/cranelift/src/gc/enabled.rs

```diff
@@ -1119,8 +1119,8 @@ impl FuncEnvironment<'_> {
         self.gc_layout(type_index).unwrap_struct()
     }
 
-    /// Get the GC heap's base pointer and bound.
-    fn get_gc_heap_base_bound(&mut self, builder: &mut FunctionBuilder) -> (ir::Value, ir::Value) {
+    /// Get the GC heap's base pointer.
+    fn get_gc_heap_base(&mut self, builder: &mut FunctionBuilder) -> ir::Value {
         let ptr_ty = self.pointer_type();
         let flags = ir::MemFlags::trusted().with_readonly();
```

Comment:
Similar to below, is this `with_readonly` going to be true in the face of a dynamic GC heap that can move in the future?